### PR TITLE
[zh-cn]: update the translation of `Permissions-Policy.autoplay` directive

### PR DESCRIPTION
**Pull request by tianyeeT**
…ctive

### CODE REVIEW

**Review by yin1999 — CHANGES_REQUESTED** [review]
https://github.com/mdn/translated-content/pull/22053#pullrequestreview-2147208712

File: files/zh-cn/web/http/headers/permissions-policy/autoplay/index.md [hunk 1]
```diff
@@ -1,26 +1,30 @@
 ---
 title: "Permissions-Policy: autoplay"
```
Comment (skyclouds2001) [inline files/zh-cn/web/http/headers/permissions-policy/autoplay/index.md:2]: 参考 https://github.com/mdn/translated-content/pull/22049#discussion_r1661735586

File: files/zh-cn/web/http/headers/permissions-policy/autoplay/index.md [hunk 2]
```diff
@@ -33,5 +37,4 @@ Permissions-Policy: autoplay <allowlist>;
 ## 参见
 
 - {{HTTPHeader("Permissions-Policy")}} 标头
-- [Permission Policy](/zh-CN/docs/Web/HTTP/Feature_Policy)
-- [使用 Permission Policy](/zh-CN/docs/Web/HTTP/Feature_Policy/Using_Feature_Policy)
+- [Permissions Policy](/zh-CN/docs/Web/HTTP/Permissions_Policy)
```
Comment (skyclouds2001) [inline files/zh-cn/web/http/headers/permissions-policy/autoplay/index.md:40]:
```suggestion
- [权限策略](/zh-CN/docs/Web/HTTP/Permissions_Policy)
```

File: files/zh-cn/web/http/headers/permissions-policy/autoplay/index.md [hunk 3]
```diff
@@ -1,26 +1,30 @@
...
+  sourceCommit: 0880a90f3811475d78bc4b2c344eb4146f25f66c
 ---
 
 {{HTTPSidebar}} {{SeeCompatTable}}
 
-The HTTP {{HTTPHeader("Permissions-Policy")}} header `autoplay` directive controls whether the current document is allowed to autoplay media requested through the {{domxref("HTMLMediaElement")}} interface. When this policy is enabled and there were no user gestures, the {{domxref("Promise")}} returned by {{domxref("HTMLMediaElement.play()")}} will reject with a `DOMException`. The [`autoplay`](/zh-CN/docs/Web/HTML/Element/audio#autoplay) attribute on {{HTMLElement("audio")}} and {{HTMLElement("video")}} elements will be ignored.
+HTTP {{HTTPHeader("Permissions-Policy")}} 标头的 `autoplay` 指令控制当前文档是否被允许通过 {{domxref("HTMLMediaElement")}} 接口自动播放媒体。
 
-For more details on autoplay and autoplay blocking, see the article [Autoplay guide for media and Web Audio APIs](/zh-CN/docs/Web/Media/Autoplay_guide).
+具体来说，如果定义的策略阻止使用此功能并且没有用户手势，{{domxref("HTMLMediaElement.play()")}} 返回的 {{jsxref("Promise")}} 将被 {{domxref("DOMException")}} 拒绝。{{HTMLElement("audio")}} 和 {{HTMLElement("video")}} 元素上的 [`autoplay`](/zh-CN/docs/Web/HTML/Element/audio#autoplay) 属性将被忽略。
```
Comment (tianyeeT) [inline files/zh-cn/web/http/headers/permissions-policy/autoplay/index.md:12]:
```suggestion
具体来说，如果定义的策略阻止使用此特性并且没有用户手势，{{domxref("HTMLMediaElement.play()")}} 返回的 {{jsxref("Promise")}} 将被 {{domxref("DOMException")}} 拒绝。{{HTMLElement("audio")}} 和 {{HTMLElement("video")}} 元素上的 [`autoplay`](/zh-CN/docs/Web/HTML/Element/audio#autoplay) 属性将被忽略。
```

File: files/zh-cn/web/http/headers/permissions-policy/autoplay/index.md [hunk 4]
```diff
@@ -1,26 +1,30 @@
...
-```
-Permissions-Policy: autoplay <allowlist>;
+```http
+Permissions-Policy: autoplay=<allowlist>;
 ```
 
-- \<allowlist>
-  - : 允许使用此特性的来源（origin）列表。参见 [`Permissions-Policy`](/zh-CN/docs/Web/HTTP/Headers/Permissions-Policy#语法)。
+- `<allowlist>`
+  - : 允许使用该功能的来源列表。请参阅 [`Permissions-Policy` > 语法](/zh-CN/docs/Web/HTTP/Headers/Permissions-Policy#语法)了解更多详细信息。
```
Comment (tianyeeT) [inline files/zh-cn/web/http/headers/permissions-policy/autoplay/index.md:23]:
```suggestion
  - : 允许使用该特性的来源列表。请参阅 [`Permissions-Policy` > 语法](/zh-CN/docs/Web/HTTP/Headers/Permissions-Policy#语法)了解更多详细信息。
```

**Review by yin1999 — COMMENTED** [review]
nits

**Review by yin1999 — APPROVED** [review]

**Pull request merged**